import/controls/permissionmodel.h

```diff
@@ -0,0 +1,57 @@
+/****************************************************************************
+**
+** Copyright (C) 2020 Open Mobile Platform LLC
+**
+****************************************************************************/
+
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef PERMISSIONMODEL_H
+#define PERMISSIONMODEL_H
+
+#include <QObject>
+#include <QAbstractListModel>
+
+struct Permission
+{
+    Permission(QString uri, QString type, QString capability)
```

Comment:
I guess this struct should depict the original [`nsIPermission`](https://developer.mozilla.org/en-US/docs/Mozilla/Tech/XPCOM/Reference/Interface/nsIPermission). If it's so then let's use the same names: `m_host` instead of `m_uri`.
Also it would be nice to leave a small comment what's type and what's capability.